CONTRIBUTING.md

```diff
@@ -177,6 +177,9 @@ All covered projects should have 70% or better test coverage.
 By default, all _Azure.*_ libraries are covered, and any project that sets the `IsClientLibrary=true` MSBuild property.
 To exclude a project, set `ExcludeFromCodeCoverage=true` in the project's MSBuild properties before other targets are imported.
 
+> The Azure SDK team does not mandate a threshold for code coverage nor do we report on it. This metric can be misleading e.g., a client library with a lot of models and few operations could have complete serialization coverage of models but no coverage for operations and still have a high metric.
```

Comment:
What does it mean that we don't report on it?  We say below that people can drill into reports, so I'm unclear on what it means to have reports and that we don't report on it.  I'm mostly just asking for clarification here.

One other note - we say above that "all covered projects should have 70% coverage or better" -- how do we reconcile this statement with the statement that we don't mandate a threshold?  It seems like we are making a recommendation without some requirement, but there's some hair-splitting here that I'm not sure I understand and might be unclear to contributors.  If we don't mandate, should we also not recommend?  I.e. should we just entirely remove the line above about 70% with this addition?

Reply:
Good catch, I read this in isolation and didn't realize there was another statement.   My vote would be that we remove the 70% statement.

Reply:
@heaths was the person you were talking to thrown off by 70% target or the number itself? I agree with the sentiment above - removing 70% or more as a recommendation\requirement would be a good first step here.

Reply:
Yeah, I forgot about that too and, like Jesse when reviewing it, basically wrote this in isolation. I'll clarify "report" (@annelo-msft I meant that we don't report that aggregate number anywhere, but I'll change verbiage so that's clear) and remove that 70% target.